packages/neo-one-client/src/user/LocalUserAccountProvider.ts

```diff
@@ -520,6 +520,44 @@ export class LocalUserAccountProvider<TKeyStore extends KeyStore, TProvider exte
       },
     });
   }
+  public async executeCompiledCode(
+    compiledCode: Buffer,
+    options: { readonly from: UserAccountID },
+  ): Promise<RawInvocationResult> {
+    const { from, attributes, networkFee, monitor } = this.getTransactionOptions(options);
+
+    return this.capture(
+      async (span) => {
+        const { inputs, outputs } = await this.getTransfersInputOutputs({
+          transfers: [],
+          from,
+          gas: networkFee,
+          monitor: span,
+        });
+
+        const testTransaction = new InvocationTransaction({
+          version: 1,
+          inputs: this.convertInputs(inputs),
+          outputs: this.convertOutputs(outputs),
+          attributes: this.convertAttributes(attributes),
+          gas: common.TEN_THOUSAND_FIXED8,
+          script: compiledCode,
+        });
+
+        return this.provider.testInvoke(from.network, testTransaction.serializeWire().toString('hex'), span);
```

Comment:
This only invokes the transaction in a test mode, i.e. it doesn't persist to the database. This method should function similar to `invoke` where the `script` argument of `invokeRaw` is the passed in script.